go-controller/pkg/ovn/address_set/address_set.go

```diff
@@ -683,13 +682,11 @@ func splitIPsByFamily(ips []net.IP) (v4 []net.IP, v6 []net.IP) {
 	return
 }
 
-func ipToStringSort(ips []net.IP) []string {
+func ipsToString(ips []net.IP) []string {
 	// my kingdom for a ".values()" function
 	out := make([]string, 0, len(ips))
 	for _, ip := range ips {
 		out = append(out, ip.String())
 	}
-	// so the tests are predictable
-	sort.Strings(out)
```

Comment:
++

Reply:
Yeah, this was only needed for ovn-nbctl when we compared command arguments directly...